<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatwindow</class>
 <widget class="QWidget" name="chatwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="chatWindowTitleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>310</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>发       送</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>249</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="sendPhotoButton">
      <property name="text">
       <string>发送图片</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sendFileButton">
      <property name="text">
       <string>发送文件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="receiveFileButton">
      <property name="text">
       <string>接收文件</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>0</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="back.qrc">
     <normaloff>:/C:/Users/24332/Desktop/MyQQ/nc.png</normaloff>:/C:/Users/24332/Desktop/MyQQ/nc.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QListWidget" name="chatList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>41</y>
     <width>181</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="ui_mag">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>41</y>
     <width>381</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="idt">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>381</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="back.qrc"/>
 </resources>
 <connections/>
</ui>
